.prettierrc

```diff
@@ -10,7 +10,5 @@
   "requirePragma": false,
   "semi": false,
   "singleQuote": true,
-  "tabWidth": 2,
```

Comment:
This corresponds to the addition of the EditorConfig-managed [`indent_size`](https://github.com/codecov/uploader/blob/0743c8b/.editorconfig#L10).

Reply:
🤦‍♀️ I thought that was this file. Too early 😹